src/test/java/com/solutions/shop/repository/CustomerRepositoryTest.java

```diff
@@ -0,0 +1,26 @@
+package com.solutions.shop.repository;
+
+import com.solutions.shop.model.Customer;
+import com.solutions.shop.model.User;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+@DataJpaTest
+public class CustomerRepositoryTest {
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Test
+    public void testCustomerCreate() {
+        User user1 = new User(998,"test-logon", "test-passw");
```

Comment:
This test tests C in CRUD. What about RUD tests?

Reply:
later  :)